en/includes/guides/organization-management/inheritance-in-organizations/index.md

```diff
@@ -19,3 +19,57 @@ The following guides explain how inheritance works for each of these features:
 - **[Email and SMS template]({{base_path}}/guides/organization-management/inheritance-in-organizations/email-sms-templates-inheritance/)**: Learn how customizations to Email and SMS templates propagate in an organization hierarchy.
 
 - **[Flow inheritance]({{base_path}}/guides/organization-management/inheritance-in-organizations/flow-inheritance/)**: Learn how organizations inherit Flows.
+
+## Upcoming Changes Effective 9th December, 2025
+
+{% if product_name == "Asgardeo" %}
+
+We are rolling out a new set of changes for sub-organizations that introduce hierarchical inheritance for the following features on the 09th of December, 2025.
+
+- Login and registration configurations
+- Connection configurations of:
+    - SMS OTP
+    - Push Notifications
+    - Email OTP
+    - Passkey connection
+- Flows
+    - Self-registration flows
+    - Password recovery flows
+    - Invited user registration flows
+- OIDC scopes
+- Attribute update verification settings
+- User attributes (claims)
+
+## Summary of Upcoming Changes
+
+- **Login & Registration Configurations and Flows**
+
+    - Previously: These configurations were not inherited by child organizations, and customization at the child level was not possible.
+
+    - After the update: Child organizations will automatically inherit login and registration configurations from their parent organization, including any designed self-registration, password recovery and invited user registration flows. Any customizations defined at the parent level will cascade down to child organizations. You will still have the flexibility to explicitly override these inherited settings at the child level if required.
+
+- **Connections**
+
+    - Previously: SMS OTP, Push Notification, Email OTP and Passkey connection configurations were not inherited by child organizations, and customizations at the child level was not possible
+
+    - After the update: Child organizations now automatically inherit these connection configurations from their parent organization. Additionally, configurations for Email OTP and SMS OTP connections can be overridden at the child organization level if needed.
```

Comment:
Since we are going with the future tense (will), let's be consistent here too. Instead of now, use will.